src/core/controllers/oidc.go

```diff
@@ -244,6 +246,84 @@ func (oc *OIDCController) Callback() {
 	}
 }
 
+func (oc *OIDCController) RedirectLogout() {
+	sessionData := oc.GetSession(tokenKey)
+	ctx := oc.Ctx.Request.Context()
+	if sessionData == nil {
```

Comment:
I don't think there it's necessary to return an error, it's possible the user is not onboarded via OIDC if the token is not in session, therefore, I also don't think you need to check the user ID in `LogOut()` in  `base.go`.

I think it's fine to move on the destroy the session if there's no token in the session.

Reply:
update logic to return to the home page on empty session.